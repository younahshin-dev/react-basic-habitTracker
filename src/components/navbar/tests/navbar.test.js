
import '@testing-library/jest-dom'
import Navbar from '../navbar';
import renderer from 'react-test-renderer';

describe('Habit', () => {

  
  const habits = [
    {id: 1, name: 'Reading', count:0},
    {id: 2, name: 'Running', count:1},
    {id: 3, name: 'Coding', count:0}
  ]; 


  it('renders', () => {
    //스냅샷 테스트
    const component = renderer.create(
      <Navbar
      totalCount={habits.filter(item => item.count > 0).length}/>
    );

    expect(component.toJSON()).toMatchSnapshot();
  });

  
    
});
