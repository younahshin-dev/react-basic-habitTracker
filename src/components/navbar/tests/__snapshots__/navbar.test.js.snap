// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Habit renders 1`] = `
<div
  className="navbar"
>
  <i
    className="navbar-logo fas fa-leaf"
  />
  <span>
    Habit Tracker
  </span>
  <span
    className="navbar-count"
  >
    1
  </span>
</div>
`;
